label_options/caebike-label-options.json

```diff
@@ -0,0 +1,85 @@
+{
+  "MODE": [
+    {"value":"walk", "baseMode":"WALKING", "met": {"ALL": {"range": [0, -1], "mets": 3}}, "kgCo2PerKm": 0},
+    {"value":"auto_driver", "baseMode":"CAR", "met_equivalent":"IN_VEHICLE", "kgCo2PerKm": 0.22031},
+    {"value":"auto_passenger", "baseMode":"CAR", "met_equivalent":"IN_VEHICLE", "kgCo2PerKm": 0.110155},
+    {"value":"e-bike", "baseMode":"E_BIKE", "met": {"ALL": {"range": [0, -1], "mets": 5}}, "kgCo2PerKm": 0.00728},
+    {"value":"bike", "baseMode":"BICYCLING", "met": {"ALL": {"range": [0, -1], "mets": 6}}, "kgCo2PerKm": 0},
+    {"value":"taxi", "baseMode":"TAXI", "met_equivalent":"IN_VEHICLE", "kgCo2PerKm": 0.30741},
+    {"value":"bike_scooter_share", "baseMode":"BICYCLING_SCOOTER_SHARE", "met": {"ALL": {"range": [0, -1], "mets": 3.5}}, "kgCo2PerKm": 0.00447},
+    {"value":"bus_train_shuttle", "baseMode":"PUBLIC_TRANSIT", "met_equivalent":"IN_VEHICLE", "kgCo2PerKm": 0.179033},
+    {"value":"other_mode", "baseMode":"OTHER_MODE", "met_equivalent":"UNKNOWN", "kgCo2PerKm": 0},
+    {"value":"not_a_trip", "baseMode":"UNKNOWN", "met_equivalent":"UNKNOWN", "kgCo2PerKm": 0}   
+  ],
+  "PURPOSE": [
+    {"value":"to_home"},
+    {"value":"to_work"},
+    {"value":"work_related"},
+    {"value":"to_school"},
+    {"value":"access_transit"},
+    {"value":"appointment_shopping_errand_meal"},
+    {"value":"pick_drop_person"},
+    {"value":"for_exercise_recreation"},
+    {"value":"to_access_recreation_entertainment_social_religious"},
+    {"value":"other_purpose"}
+  ],
+  "REPLACED_MODE": [
+    {"value":"no_travel"},
+    {"value":"walk"},
+    {"value":"auto_driver"},
+    {"value":"auto_passenger"},
+    {"value":"bike"},
+    {"value":"taxi"},
+    {"value":"bike_scooter_share"},
+    {"value":"bus_train_shuttle"},
+    {"value":"other_mode"}
+  ],
+  "translations": {
+    "en": {
+      "walk": "Walk",
+      "auto_driver": "Auto Driver",
+      "auto_passenger": "Auto Passenger",
+      "e-bike": "E-bike",
+      "bike": "Regular Bike",
```

Comment:
I have also heard "acoustic bike" recently, not sure if that is taking off as nomenclature 😄